### PR TITLE
Added  Access Log Configuration sample documentation for RESTEasy + Servlet

### PR DESCRIPTION
**Pull request by masini**
Few lines to describe how-to

### CODE REVIEW

File: docs/src/main/asciidoc/undertow-reference.adoc [hunk 1]
```diff
@@ -35,6 +35,32 @@ You can make use of the Undertow predicate language using an `undertow-handlers.
 in the `META-INF` directory of your application jar. This file contains handlers defined using the
 link:http://undertow.io/undertow-docs/undertow-docs-2.0.0/index.html#predicates-attributes-and-handlers[Undertow predicate language].
 
+=== Configuring HTTP Access Logs
+
+You can add HTTP request logging adding the AccessHandler configuration to the undertow-handler.conf file.
```
Comment (rsvoboda) [inline docs/src/main/asciidoc/undertow-reference.adoc]: `undertow-handlers.conf`

**Review by rsvoboda — CHANGES_REQUESTED** [review]
Please fix the file name

**Comment by masini** [comment]
Hi Rostislav, which file must be renamed ?

File: docs/src/main/asciidoc/undertow-reference.adoc [hunk 2]
```diff
@@ -35,6 +35,32 @@ You can make use of the Undertow predicate language using an `undertow-handlers.
...
+----
+access-log('common')
+----
+
+this will log every request to Quarkus configured logging system under the `io.undertow.accesslog` category.
+
+In case you can also configure another category such this:
+
+----
+access-log(format='common', category='my.own.category')
```
Comment (rsvoboda) [inline docs/src/main/asciidoc/undertow-reference.adoc]: Category is not recognized parameter 
```
Caused by: java.lang.RuntimeException: Failed to start quarkus
	at io.quarkus.runner.ApplicationImpl2.<clinit>(ApplicationImpl2.zig:131)
	... 16 more
Caused by: java.lang.RuntimeException: java.lang.IllegalArgumentException: UT000045: Error parsing predicated handler string unknown parameter category:
access-log(format='common', category='my.own.category')
                                                     ^
	at io.quarkus.undertow.runtime.UndertowDeploymentRecorder.bootServletContainer(UndertowDeploymentRecorder.java:405)
	at io.quarkus.deployment.steps.UndertowBuildStep$build37.deploy_0(UndertowBuildStep$build37.zig:444)
	at io.quarkus.deployment.steps.UndertowBuildStep$build37.deploy(UndertowBuildStep$build37.zig:36)
	at io.quarkus.runner.ApplicationImpl2.<clinit>(ApplicationImpl2.zig:117)
	... 16 more
Caused by: java.lang.IllegalArgumentException: UT000045: Error parsing predicated handler string unknown parameter category:
access-log(format='common', category='my.own.category')
                                                     ^
	at io.undertow.server.handlers.builder.PredicatedHandlersParser.error(PredicatedHandlersParser.java:729)
	at io.undertow.server.handlers.builder.PredicatedHandlersParser.handleHandlerNode(PredicatedHandlersParser.java:171)
	at io.undertow.server.handlers.builder.PredicatedHandlersParser.handleNode(PredicatedHandlersParser.java:114)
	at io.undertow.server.handlers.builder.PredicatedHandlersParser.parse(PredicatedHandlersParser.java:91)
	at io.undertow.server.handlers.builder.PredicatedHandlersParser.parse(PredicatedHandlersParser.java:80)
	at io.quarkus.undertow.runtime.UndertowHandlersConfServletExtension.handleDeployment(UndertowHandlersConfServletExtension.java:33)
	at io.undertow.servlet.core.DeploymentManagerImpl.handleExtensions(DeploymentManagerImpl.java:300)
	at io.undertow.servlet.core.DeploymentManagerImpl.deploy(DeploymentManagerImpl.java:155)
	at io.quarkus.undertow.runtime.UndertowDeploymentRecorder.bootServletContainer(UndertowDeploymentRecorder.java:394)
	... 19 more
```

**Comment by rsvoboda** [comment]
`undertow-handler.conf`
to
`undertow-handlers.conf`

**Comment by masini** [comment]
Ok for the filename, I forgot the final "s".

Regarding the configuration, I pasted there my current config, I also looked at the configuration builder and I read this:

```java
@Override
public Map<String, Class<?>> parameters() {

    Map<String, Class<?>> params = new 
    params.put("format", String.class);
    params.put("category", String.class);
    return params;
}
```

I'm using 0.21.1, but I can see that master version is without category.

Should I document 0.21.1 or next version (the latter I think) ?

**Comment by rsvoboda** [comment]
Maybe missing category is a bug ...
@stuartwdouglas can you look at this ?

@masini docs in master should be relevant to master, but first I think we need to know more about missing `category`

**Comment by masini** [comment]
What about this PR ?

**Comment by machi1990** [comment]
@masini looks like there a conflicts, can you rebase on top of master? Thanks. 

@rsvoboda do you mind having another look if the PR is okay for you? Thanks.

**Review by rsvoboda — DISMISSED** [review]
- This needs rebase because of onflicting files
 - Is `category='my.own.category'` working now ?
 - I still see undertow-handler.conf (missing `s` for handlers)

**Comment by cescoffier** [comment]
Rebased required.

**Comment by gsmet** [comment]
I force-pushed a rebase and some adjustments.

I _think_ the category issue was fixed in `quarkus-http`.  @rsvoboda could you see if you can still reproduce the issue?

Also, I don't know if this works without Undertow. If someone had the time to check, that would be nice.

**Comment by rsvoboda** [comment]
added comments to https://github.com/quarkusio/quarkus/issues/4059 few days ago

**Comment by rsvoboda** [comment]
category worked for `quarkus-http` based app ...

**Review by gsmet — APPROVED** [review]

**Comment by rsvoboda** [comment]
@gsmet https://github.com/quarkusio/quarkus/issues/5556
currently it has bug label, could be changed to enhancement

**Pull request merged**